playbooks/installation/custom_certificates.adoc

```diff
@@ -0,0 +1,172 @@
+---
+---
+= Configuring OpenShift to use Custom Certificates
+Red Hat Containers & PaaS CoP
+v2.0, 2016-08-18
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
+:toc: macro
+:toc-title:
+
+toc::[]
+
+_This is a followup to link:/playbooks/installation/[Installing a Highly Available OpenShift Cluster]. Many assumptions are made based on the information in that guide. If you have not yet been through this guide, we recommend doing so before continuing._
+
+== Overview
+
+OpenShift creates a lot of certificates by default. Understanding where and how those certificates are used will help to understand how it can be configured to meet your needs.
+
+In this guide we will discuss how and where TLS Certificates are used in OpenShift, how they are created, and how to go about designing a proper certificate management strategy for OpenShift. We will then show you how to configure OpenShift via Ansible to reflect this strategy.
+
+== Design of OpenShift Certificate Management
+
+OpenShift utilizes TLS Certificates in three primary ways.
+
+1. To encrypt traffic
+1. To Authenticate Certain Infrastructure Components
+1. To authenticate certain users
+  * OpenShift uses client certificates to authenticate system critical Service Accounts
```

Comment:
s/service accounts/system users and service accounts/

https://docs.openshift.com/container-platform/3.4/architecture/core_concepts/projects_and_users.html